tests/unit/src/main/scala/tests/QuickBuild.scala

```diff
@@ -141,9 +141,16 @@ case class QuickBuild(
     }
     val classpath = classDirectory :: allJars.filterNot(isSourceJar)
     val allPlugins =
-      if (ScalaVersions.isSupportedAtReleaseMomentScalaVersion(scalaVersion))
-        s"org.scalameta:::semanticdb-scalac:${V.semanticdbVersion}" :: compilerPlugins.toList
-      else compilerPlugins.toList
+      if (
+        ScalaVersions.isSupportedAtReleaseMomentScalaVersion(
+          scalaVersion
+        ) && !ScalaVersions.isScala3Version(scalaVersion)
+      ) {
+        val semanticdbVersion = BuildInfoVersions.lastSupportedSemanticdb(
+          scalaVersion
+        )
+        s"org.scalameta:::semanticdb-scalac:$semanticdbVersion" :: compilerPlugins.toList
+      } else compilerPlugins.toList
```

Comment:
Why we don't want to have `semanticdb` plugin for Scala 3 here?

Reply:
We don't have it as a plugin, it's in the compiler, so no need to download anything.